src/main/java/uk/ac/cam/cl/dtg/segue/configuration/SegueGuiceConfigurationModule.java

```diff
@@ -943,24 +946,53 @@ private static SegueJobService getSegueJobService(final PropertiesLoader propert
                     new DeleteEventAdditionalBookingInformationOneYearJob()
             );
 
+            SegueScheduledJob eventReminderEmail = SegueScheduledJob.createCustomJob(
+                "eventReminderEmail",
+                "JavaJob",
+                "Send scheduled reminder emails to events",
+                "0 0 7 * * ?",//""0 * * ? * * *",
+                Maps.newHashMap(),
+                new EventReminderEmailJob()
+            );
+
+            SegueScheduledJob eventFeedbackEmail = SegueScheduledJob.createCustomJob(
+                "eventFeedbackEmail",
+                "JavaJob",
+                "Send scheduled feedback emails to events",
+                "0 0 20 * * ?",
+                Maps.newHashMap(),
+                new EventFeedbackEmailJob()
+            );
+
             SegueScheduledJob syncMailjetUsers = new SegueScheduledSyncMailjetUsersJob(
                     "syncMailjetUsersJob",
                     "JavaJob",
                     "Sync users to mailjet",
                     "0 0 0/4 ? * * *");
 
             List<SegueScheduledJob> configuredScheduledJobs = new ArrayList<>(Arrays.asList(PIISQLJob, cleanUpOldAnonymousUsers,
-                    cleanUpExpiredReservations, deleteEventAdditionalBookingInformation, deleteEventAdditionalBookingInformationOneYearJob));
+                    cleanUpExpiredReservations));
 
             if (mailjetKey != null && mailjetSecret != null) {
                 configuredScheduledJobs.add(syncMailjetUsers);
             }
 
+            if (eventPrePostEmails != null) {
```

Comment:
Maybe this should be `eventPrePostEmails != null && Boolean.parseBoolean(eventPrePostEmails);`? Then we can leave the key in but set it to false too, and if set it has to be explicitly `true` or `false`  for the API to even start at all.